tmt/utils.py

```diff
@@ -6120,30 +6122,40 @@ def regex_search(
     return environment
 
 
-def render_template_file(
-        template_filepath: Path,
+def render_template(
+        template_str: str,
```

Comment:
Unless there's a strong opinion behind the `_str` suffix, I'd recommend simple `template`. `template_str: str` - one of the "str" is repeating the other one. A template is a string, with some characters having special meanings, so I'd avoid adding "str" and "string" when talking about "template". E.g. "Render a template string" => "Render a template".
`</bikeshedding>`

Reply:
Renamed in 78d7217caecb8c3bd05d92445264de9de4bfaf02.